core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1688,4 +1689,22 @@ public function generateRecordLabel(array $row, string $table = null, bool $prot
 
 		return $label;
 	}
+
+	protected function loadActiveRecord($id = null): void
```

Comment:
having this in `DataContainer` assumes they all have a database, not sure this makes sense? It also seems unused in `DC_Folder`, maybe it should be moved to `DC_Table` instead?

Reply:
It's protected so you can override it.

Reply:
The active record variable itself has resided in `DataContainer` forever so it should remain here :)